<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardDialog</class>
 <widget class="QDialog" name="WizardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wizard - SVG Cleaner</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxMain">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="horizontalSpacing">
           <number>-1</number>
          </property>
          <item row="11" column="0">
           <widget class="QGroupBox" name="gBoxCompress">
            <property name="title">
             <string>Compress</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="0" colspan="3">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Use compressor:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cmbBoxCompressor">
                 <item>
                  <property name="text">
                   <string notr="true">gzip</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string notr="true">7z</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBoxCompress">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>9</number>
                 </property>
                 <property name="value">
                  <number>9</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Compress level:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <widget class="QRadioButton" name="rBtnCompressAll">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Compress all to svgz</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QRadioButton" name="rBtnSaveSuffix">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Use input file suffix</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Open files from the input folder and:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QFrame" name="frameInDir">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <property name="horizontalSpacing">
              <number>-1</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item row="1" column="1">
              <widget class="LineEdit" name="lineEditInDir">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="2" colspan="2">
              <widget class="QToolButton" name="btnOpenInDir">
               <property name="text">
                <string notr="true">...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lblInFolder">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Input folder:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QFrame" name="frameOutDir">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>-1</number>
             </property>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditOutDir"/>
             </item>
             <item row="1" column="2">
              <widget class="QToolButton" name="btnOpenOutDir">
               <property name="text">
                <string notr="true">...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Output folder:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QFrame" name="frameRename">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>-1</number>
             </property>
             <item row="3" column="0">
              <widget class="QLineEdit" name="lineEditPrefix">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QLabel" name="label_7">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string notr="true">.svg</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="lineEditSuffix">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="lblFileName">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>filename</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="4">
              <widget class="QLabel" name="lblExample">
               <property name="text">
                <string>Example: </string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="4">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Set prefix and suffix:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="chBoxRecursive">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Recursive folder scanning</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioBtn1">
               <property name="accessibleName">
                <string notr="true">1</string>
               </property>
               <property name="text">
                <string>save cleaned files into output folder</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioBtn2">
               <property name="accessibleName">
                <string notr="true">2</string>
               </property>
               <property name="text">
                <string>save cleaned files into current folder</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioBtn3">
               <property name="accessibleName">
                <string notr="true">3</string>
               </property>
               <property name="text">
                <string>overwrite original files</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="9" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Select preset:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2" colspan="6">
           <widget class="QTextBrowser" name="textPresetInfo">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="html">
             <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Verdana'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cmbBoxPreset">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Save current settings as preset:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="linePresetName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="btnRmPreset">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4" colspan="2">
           <widget class="QPushButton" name="btnSavePreset">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>gBoxCompress</tabstop>
  <tabstop>cmbBoxCompressor</tabstop>
  <tabstop>spinBoxCompress</tabstop>
  <tabstop>rBtnCompressAll</tabstop>
  <tabstop>rBtnSaveSuffix</tabstop>
  <tabstop>lineEditInDir</tabstop>
  <tabstop>btnOpenInDir</tabstop>
  <tabstop>lineEditOutDir</tabstop>
  <tabstop>btnOpenOutDir</tabstop>
  <tabstop>lineEditPrefix</tabstop>
  <tabstop>lineEditSuffix</tabstop>
  <tabstop>chBoxRecursive</tabstop>
  <tabstop>radioBtn1</tabstop>
  <tabstop>radioBtn2</tabstop>
  <tabstop>radioBtn3</tabstop>
  <tabstop>textPresetInfo</tabstop>
  <tabstop>cmbBoxPreset</tabstop>
  <tabstop>linePresetName</tabstop>
  <tabstop>btnRmPreset</tabstop>
  <tabstop>btnSavePreset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
